### DISCUSSION TITLE
Support PAPI comparison

### DISCUSSION THREAD

**Discussion by heymoan** [Zrips/CMI#5064] [Feature Requests]
I hope the plugin can support a function
like this:

CustomAlias:
  buy:
  - Papi:[%playerpoints% >= 100]
  2Buy:
  - Papi:[%playerjob% == assassin]

English not good ,i think You know what I mean.

**Comment by CloudeLecaw** [comment]
no not realy
the plugin support allready many placeholder

so may you descripe what your trying to do here?
Check if someone is Assasin - where?
Check if someone has Playerpoint - where?

CMI allready support PlaceholderAPI (Papi)
So if a placeholder works there (and out of the plugin u use) - the chance is good that cmi can use is.

For others.. CMI allready can do Equal and greater,lesser then calculations with suportet placeholders

Example:
` - check:%cmi_user_name%==Cloude_Lecaw! cmi broadcast !Cloude_Lecaw is here. &4&l❤`
or
`- check:%cmi_user_balance_formated%>=100000! cmi ctellraw [playerName]<T>Your a rich man</T>`
or
`check:%cmi_user_group%=Owner! cmi money give all 1000`
or
`check:%worldborder_center_x%!>=10000! cmi ctellraw [playerName] this world is long.`
`check:%worldborder_center_z%!>=10000! cmi ctellraw [playerName] this world is high.`
or
`check:%player_total_exp%>=1! cmi ctellraw [playerName] Wow you have more than one XP!!!
`

so im sure u can do something with your playerjob or playerpoints if they are correct supported by Papi

**Comment by CloudeLecaw** [comment]
u know request here got ignored by the devs mostly? This here is jsut for issues
send them here aggain: https://github.com/Zrips/CMI/discussions
